inst/shiny/www/style.css

```
@@ -19,6 +19,49 @@
   width: 100%;
 }
 
+/* column mapping UI */
+
+.column-mapping-row {
+  display: flex;
+  align-items: center;
+  margin-top: -1rem;
+  margin-bottom: -1.5rem;
+}
+
+.column-mapping-output {
+  margin-left: 0.625rem;
+  margin-top: 1.25rem;
+  margin-bottom: 0;
```

nitpick: This should work fine:
```suggestion
margin: 1.25rem 0 0 0.625rem;
```